starter_code/script.js

```diff
@@ -1,9 +1,95 @@
 window.onload = function() {
   document.getElementById("start-button").onclick = function() {
-    startGame();
+    myGameArea.start();
+    myGameArea.makeRoad();
   };
 
-  function startGame() {
+  // function updateGameArea() {
+  //   myGameArea.clear();
+  //   player.update();
+} 
 
+  var myGameArea = {
+    canvas : document.getElementById("canvas"),
+    start : function() {
+        this.width = 500;
+        this.height = 600;
+        this.context = this.getContext("2d");
```

Comment:
My browser has an issue with this line. You should do something like "document.getElementById("canvas").getContext("2d")"